src/test/groovy/org/prebid/server/functional/tests/BaseSpec.groovy

```diff
@@ -89,4 +90,12 @@ abstract class BaseSpec extends Specification implements ObjectMapperWrapper {
             }]
         }
     }
+
+    protected static Map<String, List<BidderRequest>> getRequest(AmpResponse ampResponse) {
```

Comment:
Naming. Along with the previous method. It's not singular request. This method returns multiple requests. Hence name should reflect this.
`getRequest()` -> `getRequests()` for both occurrences please.